### ISSUE TITLE
Enhancement: solid colors with alpha channel in global styles palette

### ISSUE THREAD

**Issue by ghost** [WordPress/gutenberg#42999]

**Comment by colorful-tones** [comment]
@marceloaof - I would say we should close this out. It is a duplicate of #43041 and the other Issue has far more detail and context. 🤔😉

**Comment by marceloaof** [comment]
Yes, I also appreciate how the other person brought her issue... with enough details and context as I did. These tickets are precisely related and not the same. 😉 The scenario of 42999 is less specific than 43041. In 43041, colors with alpha may already have been populated by the theme, as seen in screenshots, they just are not editable. In 42999, colors with alpha are not even set yet. The solution to one may not solve the other.

If they were the same issue, it's strange to imagine that duplicate is what came before.🤔
This is the opposite meaning of the word.
A legacy is not a duplication of the present.

You meant that there will likely be a solution for 43041 which will also resolve 42999 here... in this case, I remain waiting to know whether or not 42999 ticket has been considered.
Meanwhile, there is no valid reason to close it.

As I expanded on your comment with far more detail and context, also fixing the meaning of words, feel free to delete it. Now your comment is a duplicate of mine.
